content/docs/alerting/notifications.md

```diff
@@ -91,3 +91,4 @@ templating.
 | reReplaceAll | pattern, replacement, text | [Regexp.ReplaceAllString](http://golang.org/pkg/regexp/#Regexp.ReplaceAllString) Regexp substitution, unanchored. |
 | join | sep string, s []string | [strings.Join](http://golang.org/pkg/strings/#Join), concatenates the elements of s to create a single string. The separator string sep is placed between elements in the resulting string. (note: argument order inverted for easier pipelining in templates.) |
 | safeHtml | text string | [html/template.HTML](https://golang.org/pkg/html/template/#HTML), Marks string as HTML not requiring auto-escaping. |
+| stringSlice | variadic string | returns the variadic string as a slice of string |
```

Comment:
I think string... would be clearer.

Returns the passed strings as a slice of strings.

Reply:
ok, I modified it